src/Features/Core/Portable/EditAndContinue/DebuggingSession.cs

```diff
@@ -295,6 +295,15 @@ public bool TryGetOrCreateEmitBaseline(Project project, out ImmutableArray<Diagn
             return true;
         }
 
+        /// <summary>
+        /// Gets the capabilities of the runtime with respect to applying code changes.
+        /// </summary>
+        public async Task<ManagedEditAndContinueCapabilities> GetCapabilitiesAsync(CancellationToken cancellationToken)
+        {
+            var runtimeCapabiltiies = await DebuggerService.GetCapabilitiesAsync(cancellationToken).ConfigureAwait(false);
```

Comment:
> DebuggerService.GetCapabilitiesAsync(cancellationToken).ConfigureAwait(false); [](start = 43, length = 79)

I think it's fine for now to call this in StartDebuggingSessionAsync and store the resulting flags in a field of DebuggingSession

Reply:
Yeah, I was going to do this, but being async made it a little annoying to just grab the value in the DebuggingSession constructor. I'll pass it in, and make the call sites async.